src/index.html

```diff
@@ -1,14 +1,68 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="container">
+      <div class="logo"></div>
+      <nav class="menu">
+        <a
+        href=""
+        class="menu-button is-active"
+        >
+          APPLE
+          <div class="is-active-bottom"></div>
+        </a>
+        <a
+          href=""
+          class="menu-button"
+        >
+          SAMSUNG
+        </a>
+        <a
+          href=""
+          class="menu-button"
+        >
+          СМАРТФОНЫ И ТЕЛЕФОНЫ
+        </a>
+        <a
+          href=""
```

Comment:
```suggestion
          href="#"
```
не оставляй href пустым